node/linux/datadog-aas-node-linux/datadog_wrapper

```diff
@@ -1,44 +1,76 @@
-#!/usr/bin/env sh 
+#!/usr/bin/env sh
+
+CURRENT_DIR=$(pwd)
+
+if [ -z "$DD_DIR" ]; then
+    DD_DIR="/home/datadog"
+fi
+
+if [ -z "$DD_BINARIES_URL" ]; then
+    DD_BINARIES_URL="https://gist.github.com/jcstorms1/7e73ab0da121f42d1843dac5b75c21cb/raw"
+fi
+
+setUpCommonEnv() {
+    echo "Creating datadog directory"
+    mkdir -p $DD_DIR && cd $DD_DIR || return
+
+    echo "Downloading required datadog binaries from $DD_BINARIES_URL"
+    curl -L $DD_BINARIES_URL/datadog.tar.gz -o datadog.tar.gz || return
+
+    echo "Decompressing files"
+    tar -xzf datadog.tar.gz || return
+    rm datadog.tar.gz
+
+    echo "Starting trace agent"
+    ./trace-agent &
+    TRACER_PID=$!
+    trap 'kill $TRACER_PID ; exit' INT
```

Comment:
This is required, else trying to stop this script will hang forever as it waits for the trace agent to stop.